### ISSUE TITLE
封面中标题以及其他中文内容无法加粗问题

### ISSUE THREAD

**Issue by xdedm11** [a171232886/TJUThesis_master_2021#21]
论文封面要求大多数标题以及说明都是加粗的，经过查找latex文件确认了格式是设置为加粗的。但经过测试，只有英文内容可以加粗，中文内容不可以，有遇到同样问题的校友可以交流一下

**Comment by xdedm11** [comment]
解决方案来了，找到format.tex文件，一开始的注释作者就给出了解决办法，照这个操作就好。

**Comment by a171232886** [comment]
> 解决方案来了，找到format.tex文件，一开始的注释作者就给出了解决办法，照这个操作就好。

答案就在写在里面，想不到吧。（坏笑脸.jpg）

**Comment by xdedm11** [comment]
> 

他也没说是这种问题的解决方案啊（暴风哭泣

**Comment by a171232886** [comment]
> 

难道这就是无心插柳柳成荫么。。。

我在第一次上传模板的时候，确实测试了Overleaf和Texlive，但是后来的改动都是基于Overleaf上做的，也就是ubuntu的编译环境。所以，如果遇到了一些奇怪的编译问题，改成Overleaf大概率能解决。。。

**Comment by xdedm11** [comment]
> > 
> 
> 难道这就是无心插柳柳成荫么。。。
> 
> 我在第一次上传模板的时候，确实测试了Overleaf和Texlive，但是后来的改动都是基于Overleaf上做的，也就是ubuntu的编译环境。所以，如果遇到了一些奇怪的编译问题，改成Overleaf大概率能解决。。。

是的，测试的时候用overleaf可以正常加粗，但是我现在的大论文太长了，在overleaf上直接编译超时，所以只能配置本地环境了

**Comment by a171232886** [comment]
> 是的，测试的时候用overleaf可以正常加粗，但是我现在的大论文太长了，在overleaf上直接编译超时，所以只能配置本地环境了

虽然不知道你是硕士还是博士，但我硕士36000字的论文都是在overleaf上写的。编译超时更可能的原因是overleaf是国外网站。。。

**Comment by Liguanding** [comment]
没懂，开头的注释不是对应于特殊字体嘛，换字体解决中文不能加粗？

**Comment by xdedm11** [comment]
> 

是的。不过这个封面是老版的，想提交图书馆建议去 http://www.lib.tju.edu.cn/show.jsp?classid=201804071432181782&informationid=201804112256534890 下载最新版的封面

**Comment by Liguanding** [comment]
> 
是的，还想改以下\makebox 来着，不过看了半天实在不知道怎么把单独一行的一个box宽度调大，还是单独搞封面吧。感谢！
